exercises/acronym/example.rs

```diff
@@ -0,0 +1,27 @@
+///
+/// Techies love their TLA (Three Letter Acronyms)!
```

Comment:
I think it is possible this can get removed from the example, since it won't get shown to students and this information is in the README already.